changelog/19.0/19.0.0/summary.md

```diff
@@ -6,6 +6,8 @@
   - **[Deprecations and Deletions](#deprecations-and-deletions)**
   - **[Docker](#docker)**
     - [New MySQL Image](#mysql-image)
+  - **[VTGate](#vtgate)**
```

Comment:
Let's mark the header as Query Compatibility. 
We can add more query support in this section when added.

Reply:
Done